mne/beamformer/_lcmv.py

```diff
@@ -266,7 +337,7 @@ def lcmv(evoked, forward, noise_cov, data_cov, reg=0.05, label=None,
     forward : dict
         Forward operator
     noise_cov : Covariance
-        The noise covariance
+        The noise covariance. If provided, whitening will be done.
```

Comment:
add

"Provided a noise covariance is mandatory if you mix sensor types, e.g. gradiometers with magnetometers or EEG with MEG."